lib/ui/page/home/page/chat/widget/media_attachment.dart

```diff
@@ -28,6 +28,7 @@ import '/ui/page/home/page/chat/widget/video_thumbnail/video_thumbnail.dart';
 import '/ui/page/home/widget/retry_image.dart';
 import '/ui/widget/svg/svg.dart';
 import '/ui/worker/cache.dart';
+import 'AudioPlayer.dart';
```

Comment:
https://dart.dev/effective-dart/style#do-name-packages-and-file-system-entities-using-lowercase-with-underscores